10_intermediate/clean.py

```diff
@@ -0,0 +1,139 @@
+import pandas as pd
+
+url = "/Users/lorna/Downloads/prescription_data.zip"
+
```

Comment:
probably add a comment here: "these chosen 12 states include three states of interest and 3 constatnt states for each concerning state respectively, which is derived from the state_selection function "